ga4gh/cli.py

```diff
@@ -270,7 +380,17 @@ def client_main():
         help="Benchmark server performance")
     bmParser.set_defaults(runner=BenchmarkRunner)
     addUrlArgument(bmParser)
-    addOptions(bmParser)
+    addVariantSearchOptions(bmParser)
+
+    # variants/search
+    vsParser = subparsers.add_parser(
```

Comment:
It might improve readability if we just use `subParser` for each of these subcommands (instead of `bmParser`, `vsParser`, etc). It gets a bit confusing trying to come up with names for each one, and we're not actually using these variables again.